pkg/apiserver/handlers/webhook/mutation_labels.go

```diff
@@ -0,0 +1,179 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webhook
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	admv1 "k8s.io/api/admission/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+)
+
+type jsonPatchOperation string
+
+const (
+	jsonPatchReplaceOp jsonPatchOperation = "replace"
+	// LabelMetadataName is a well known reserved label key used by Antrea to store the resource's name
+	// as a label value.
+	LabelMetadataName = "antrea.io/metadata.name"
+)
+
+// jsonPatch contains necessary info that MutatingWebhook required
+type jsonPatch struct {
+	// Op represent the operation of this mutation
```

Comment:
s/Op represent/Op represents

Reply:
Will fix this in one of my subsequent PRs